### ISSUE TITLE
Showing compilation time %

### ISSUE THREAD

**Issue by anandijain** [KristofferC/TimerOutputs.jl#114]
`@time` returns `0.014799 seconds (7.36 k allocations: 290.106 KiB, 98.36% compilation time)`

I'm wondering if TimerOutputs could also show the compilation %, or if there is a reason it can't

Thanks again Kristoffer!

**Comment by IanButterworth** [comment]
This would be nice. Perhaps also/alternatively separate out the first run of each item? The `avg` can be thrown off significantly by compilation time. 

It could be presented as `first_run` and `warm_avg` where the latter is the average of `runs[2:end]` ?

**Comment by KristofferC** [comment]
I don't think you can really measure this because you don't know how much Julia compiles down into your "lower" section.

**Comment by IanButterworth** [comment]
I'm not sure that's true? These numbers seem reasonable. Perhaps this test is too simple though
```julia
@time begin
    x = rand(10, 10);
    x * x
    @time begin
        y = rand(10, 10)
        y / y
    end
end
````

```
  1.062033 seconds (2.65 M allocations: 146.887 MiB, 4.21% gc time, 99.54% compilation time)
  1.918953 seconds (5.07 M allocations: 276.690 MiB, 5.88% gc time, 98.69% compilation time)
```
